coalib/output/Logging.py

```diff
@@ -1,28 +1,73 @@
 from datetime import datetime
-import json
 import io
+import json
 import logging
 import logging.config
 
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
+
 
-def configure_logging():
+def configure_logging(new_log_level=logging.INFO, incremental=False,
+                      stdout=False, color=True):
     """
     Configures the logging with hard coded dictionary.
+    The incremental param is ``True`` the log-handlers configuration
+    other than level is ignored. Thus, use it when changing the log_level
+    json-handler are created so that they exist when
+    the Log Level is changed via ``configure_logging``.
+
+    :param new_log_level: The new log level to set for json and console
+                          handlers.
+    :param incremental: ``False`` if setting up new loggers, ``True`` if only
+                        changing the log level.
+    :param stdout: If ``True`` the logging is done on stdout, else its done
+                   stderr.
+    :param color: If ``True`` the logging uses ``color`` formatter, else it uses
+                  the ``no-color`` formatter.
+    :return: The new_log_level is returned.
+
+    >>> from coalib.output.Logging import configure_logging
+    >>> import logging
+    >>> import sys
+    >>> configure_logging(stdout=True, color=False)
+    20
+    >>> logging.info("This world doesn’t belong to them")
+    [INFO]... This world doesn’t belong to them
+
+    We will now change the level to ERROR.
```

Comment:
I would generally add a newline before a code sample, makes it easier to distinct